zerver/openapi/zulip.yaml

```diff
@@ -960,10 +960,14 @@ paths:
                                 }
                             - type: object
                               description: |
-                                A simple event sent to organization administrators when the
-                                set of invitations changes; this tells clients they need to refetch
+                                A simple event sent when the set of invitations changes.
+                                This event is sent to organization administrators and creator of
+                                changed invitation; this tells clients they need to refetch
```

Comment:
Needs to be "the creator of the changed invitation" because we're referring to a specific user and invitation. Might be nice to expand the first sentence for clarity too...

> A simple event sent when the set of invitations in the organization changes. This event is sent to organization administrators and the creator of the changed invitation; this tells clients ...

And in the **Changes** note below, maybe expand to "organization administrators" for consistency with the text above.